documentation/modules/ref-operator-cluster.adoc

```diff
@@ -121,6 +118,30 @@ The DNS domain name is added to the Kafka broker certificates used for hostname
 +
 If you are using a different DNS domain name suffix in your cluster, change the `KUBERNETES_SERVICE_DNS_DOMAIN` environment variable from the default to the one you are using in order to establish a connection with the Kafka brokers.
 
+[discrete]
+== Logging configuration
+
+A `ConfigMap` containing logging configuration is created when installing the Cluster Operator.
+This `ConfigMap` is described in the file `install/cluster-operator/050-ConfigMap-strimzi-cluster-operator-logging.yaml`.
+You configure Cluster Operator logging by changing the `ConfigMap`.
+
+To update the logging configuration, you can edit the `050-ConfigMap-strimzi-cluster-operator-logging.yaml` file and then run the following command:
```

Comment:
I think you should mention that the logging configuration is in the `log4j2.properties` data field.
If in the future we will use the `ConfigMap` for something more, we will have more data fields.